sparql/sparql11/protocol/manifest.ttl

```diff
@@ -1,961 +1,1430 @@
-    
-#### Response
-
-    4xx
-       """ ;
-       dawgt:approval dawgt:Approved ;
-       dawgt:approvedBy <http://www.w3.org/2009/sparql/meeting/2012-11-20#resolution_3> ;
-       .
+  }
+}
+"""
+        ];
+        ht:headers ([
+            a ht:RequestHeader;
+            ht:fieldName "Accept";
+            ht:fieldValue "application/sparql-results+xml";
+            ht:headerElements ([
+                a ht:HeaderElement;
+                ht:elementName "application/sparql-results+xml"
+              ])
+          ] [
+            a ht:RequestHeader;
+            ht:fieldName "Content-Type";
+            ht:fieldValue "application/sparql-query";
+            ht:headerElements ([
+                a ht:HeaderElement;
+                ht:elementName "application/sparql-query"
+              ])
+          ]);
+        ht:httpVersion "1.1";
+        ht:methodName "POST";
+        ht:resp [
+          a ht:Response;
+          ht:headers ([
+              a ht:RequestHeader;
+              ht:fieldName "Content-Type";
+              ht:fieldValue "application/sparql-results+xml";
+              ht:headerElements ([
+                  a ht:HeaderElement;
+                  ht:elementName "application/sparql-results+xml"
+                ])
+            ]);
+          ht:statusCodeValue "2XX",
+            "3XX"
+        ]
+      ])
+  ];
+  mf:name "update with protocol-specified named graphs" .
+
+<#update_post_direct> a mf:ProtocolTest;
+  dawg:approval dawg:Approved;
+  dawg:approvedBy <http://www.w3.org/2009/sparql/meeting/2012-11-20#resolution_3>;
+  mf:action [
+    a ht:Connection;
+    ht:connectionAuthority "www.example";
+    ht:requests ([
+        a ht:Request;
+        ht:absolutePath "/sparql";
+        ht:body [
+          a cnt:ContentAsText;
+          cnt:characterEncoding "UTF-8";
+          cnt:chars "CLEAR ALL"
+        ];
+        ht:headers ([
+            a ht:RequestHeader;
+            ht:fieldName "Content-Type";
+            ht:fieldValue "application/sparql-update";
+            ht:headerElements ([
+                a ht:HeaderElement;
+                ht:elementName "application/sparql-update"
+              ])
+          ]);
+        ht:httpVersion "1.1";
+        ht:methodName "POST";
+        ht:resp [
+          a ht:Response;
+          ht:statusCodeValue "2XX",
+            "3XX"
+        ]
+      ])
+  ];
+  mf:name "update via POST directly" .
+
+<#update_post_form> a mf:ProtocolTest;
+  dawg:approval dawg:Approved;
+  dawg:approvedBy <http://www.w3.org/2009/sparql/meeting/2012-11-20#resolution_3>;
+  mf:action [
+    a ht:Connection;
+    ht:connectionAuthority "www.example";
+    ht:requests ([
+        a ht:Request;
```

Comment:
Request is missing `Content-Type: application/x-www-url-form-urlencoded` header.